indico/MaKaC/PDFinterface/conference.py

```diff
@@ -2190,8 +2190,8 @@ def firstPage(self, c, doc):
         )
         config = Config.getInstance()
         baseURL = config.getBaseSecureURL() if config.getBaseSecureURL() else config.getBaseURL()
-        qr_data = {"registrant_id": self._registrant.getId(),
-                   "checkin_secret": self._registrant.getCheckInUUID(),
+        qr_data = {"registrant_id": self._registration.friendly_id,
```

Comment:
should be `self._registration.id` - friendly_id is only used when showing it to users, but not in technical contexts